app/core/crossref/book.tsx

```diff
@@ -18,22 +52,22 @@ const book = ({
   doi,
   resolve_url,
   citations,
-}) => {
-  const js = {
+}: BookProps) => {
+  const js: Book = {
     type: "element",
     name: "book",
     attributes: {
       book_type: "other",
     },
-    elements: [
+    children: [
       {
         // https://data.crossref.org/reports/help/schema_doc/4.4.2/schema_4_4_2.html#book_metadata
         type: "element",
         name: "book_metadata",
         attributes: {
-          language: "en",
+          language,
```

Comment:
```suggestion
          language: "en",
```

Needs to be hardcode, otherwise this is updated every time somebody submits in a different language 😅

Reply:
Ahhhhh right, I didn't think about the "everything is a chapter" that deeply but yeah that's not ideal haha. I think I changed that because I had a merge conflict at one point (you can check the merge commit) where I think the language was added as an argument, but that was probably for the chapter 😅